setup.py

```diff
@@ -27,15 +27,17 @@ def read(filename):
     author='MAAS Developers',
     author_email='maas-devel@lists.launchpad.net',
     url='https://github.com/maas/python-libmaas',
-    version="0.6.6",
+    version="0.6.7",
```

Comment:
Could you please revert the version bump? We usually do it as a separate commit before releasing.

Reply:
Done